search-ui/tests/unit/Breadcrumb.spec.ts

```diff
@@ -0,0 +1,54 @@
+// Libraries
+import { RouteNames } from '@/enums'
+import { mount } from '@vue/test-utils'
+import { BreadcrumbIF } from '@bcrs-shared-components/interfaces'
+import { BcrsBreadcrumb } from '@/bcrs-common-components'
+import { SearchDashboardBreadcrumb, SearchHomeBreadCrumb } from '@/bcrs-common-components/resources'
+import store from '@/store'
+
+
+describe('Breadcrumb.vue', () => {
+
+  const breadcrumbs : BreadcrumbIF[] = [
+    {
+      text: SearchHomeBreadCrumb.text,
+      href: "https://yfthig-dev.web.app/dashboard"
+    },
+    {
+      text: SearchDashboardBreadcrumb.text,
+      to: { name: RouteNames.DASHBOARD }
+    }
+  ]
+
+  const wrapper = mount(BcrsBreadcrumb, {
+    props: { breadcrumbs },
+    global: {
+      provide: {
+        store: store
+      }
+    },
+    shallow: true  // stubs out children components
+  })
+
+  it('present page is dashboard so should display breadcrumb', () => {
+    expect(wrapper.exists()).toBe(true)
+    expect(wrapper.find('.v-breadcrumbs').exists()).toBe(true)
+    const breadcrumbItems = wrapper.findAll('.v-breadcrumb-item')
+    expect(breadcrumbItems.length).toBe(breadcrumbs.length)
+    for (let i = 0; i < breadcrumbs.length; i++) {
+      expect(wrapper.find('.v-breadcrumbs').html()).toContain(breadcrumbs[i].text)
+    }
+  })
+
+  it('dashboard back should be redirect to search online service page', () => {
+    expect(wrapper.vm.backUrl()).toBe("https://yfthig-dev.web.app/dashboard")
```

Comment:
pls update to expect(wrapper.vm.backUrl()).toBe(breadcrumbs[0].href) or set both to a global var so that if we alter that above this test will also get altered.

In the future it would be better if you imported the actual router and set the session storage value of registry url as a variable and then verified that variable here (don't worry about that for now).

Reply:
Updated. Thanks!